### ISSUE TITLE
Bug Report

### ISSUE THREAD

**Issue by FamilyGuy0395** [FlagBrew/Checkpoint#278]
@BernardoGiordano I recently opened up a bug report and BernardoGiordano told me that formatting my microSD card to FAT32 would solve my problems.  My microSD card is now properly formatted and I'm still experiencing the same issues.

See post below

**Comment by GriffinG1** [comment]
(Prior was #274)

**Comment by FamilyGuy0395** [comment]
> (Prior was #274)

Thanks Griffin!

**Comment by FamilyGuy0395** [comment]
For some reason now I'm able to backup my saves, but I'm still unable to view the saves that I had made prior to formatting my SD card from exFAT to FAT32.

**Comment by BernardoGiordano** [comment]
Can you show me where did you place said backups on your SD card?

**Comment by FamilyGuy0395** [comment]
@BernardoGiordano 

Under switch --> Checkpoint --> saves, Checkpoint creates individual folders for each game.  For example, the folder for Pokémon Let's Go, Pikachu! is "0x010003F003A34000 0x010003F003A34000" (frustratingly, some games don't have the actual game name listed next to it, which I really wish would be fixed).  Inside these game folders, are the individual save folders that contain the actual save files inside.  Thus, after I formatted my SD card, I re-created a save file for each and every one of my games so Checkpoint would create the individual game folders.  After that, I then copy and pasted the my old save file folders into their respective game folders, and Checkpoint doesn't recognize any save file folders prior to my formatting.

TL;DR I am now able to backup my saves and restore my saves.  However, after correctly copying and pasting my old save files into their respective game folders, Checkpoint is unable to recognize those save files at all, and thus I can't view them nor restore them within Checkpoint on the Switch.
